packages/nodes-base/nodes/EmailReadImap/v2/EmailReadImapV2.node.ts

```diff
@@ -170,6 +176,22 @@ const versionDescription: INodeTypeDescription = {
 					default: 60,
 					description: 'Sets an interval (in minutes) to force a reconnection',
 				},
+				{
+					displayName: 'Message Limit',
+					name: 'messageLimit',
+					type: 'number',
+					default: 50,
+					description:
+						'Maximum number of messages to fetch (0 for unlimited). Fetches the most recent messages first.',
+					typeOptions: {
+						minValue: 0,
+					},
+					displayOptions: {
+						show: {
+							'@version': [{ _cnd: { gte: 2.1 } }],
+						},
+					},
+				},
```

Comment:
we need to make user aware of consequences of enabling this option described [here](https://linear.app/n8n/issue/NODE-2648/community-issue-problem-with-node-email-trigger-imap#comment-c5441772)

since it is enabled by default we may want to add hint to output pane to make user aware of it